packages/pygobject2.rb

```diff
@@ -24,7 +24,7 @@ class Pygobject2 < Package
 
   depends_on 'glib'
   depends_on 'gobject_introspection'
-  depends_on 'pycairo'
+  depends_on 'py3_pycairo'
```

Comment:
We need a py2_pycairo, since pygobjects2 is the python2 version of pygobjects.

(This is needed for nmap)

```
# Adapted from Arch Linux nmap PKGBUILD at:
# https://github.com/archlinux/svntogit-packages/raw/packages/nmap/trunk/PKGBUILD

require 'package'

class Nmap < Package
  description 'Utility for network discovery and security auditing'
  homepage 'https://nmap.org/'
  version '7.91'
  compatibility 'all'
  license 'GPL2'
  source_url 'https://nmap.org/dist/nmap-7.91.tar.bz2'
  source_sha256 '18cc4b5070511c51eb243cdd2b0b30ff9b2c4dc4544c6312f75ce3a67a593300'

  depends_on 'libpcap'
  depends_on 'libssh2'

  def self.build
    # ensure we build devendored deps"
    @deps = %w[libpcap libpcre macosx mwin32 libssh2 libz]
    @deps.each do |dep|
      FileUtils.rm_rf dep if Dir.exist?(dep)
    end
    system 'autoreconf -fiv'
    system 'filefix'
    system "./configure --help"
    system "env #{CREW_ENV_OPTIONS} \
    ./configure \
    #{CREW_OPTIONS} \
    --with-libpcap=#{CREW_PREFIX} \
    --with-libpcre=#{CREW_PREFIX} \
    --with-zlib=#{CREW_PREFIX} \
    --with-libssh2=#{CREW_PREFIX} \
    --with-liblua=included \
    --without-ndiff"
    system 'make'
    @zenmap_ = <<~'ZENMAP_EOF'
      #!/bin/sh
      xhost si:localuser:root
      sudo -E LD_LIBRARY_PATH=#{CREW_LIB_PREFIX} zenmap.elf
    ZENMAP_EOF
    IO.write('zenmap_', @zenmap_)
  end

  def self.install
    system "make DESTDIR=#{CREW_DEST_DIR} install"
    FileUtils.mv "#{CREW_DEST_PREFIX}/bin/zenmap", "#{CREW_DEST_PREFIX}/bin/zenmap.elf"
    FileUtils.install 'zenmap_', "#{CREW_DEST_PREFIX}/bin/zenmap", mode: 0755
  end
end
```

Reply:
(That version of nmap just got merged, but it still needs python2 libraries rebuilt...)